src/pip_sync_faster/cli.py

```diff
@@ -0,0 +1,28 @@
+from argparse import ArgumentParser
+from importlib.metadata import version
+from subprocess import CalledProcessError
+
+from pip_sync_faster.sync import sync
+
+
+def cli(_argv=None):  # pylint:disable=inconsistent-return-statements
```

Comment:
PyLint will force you to put `return 0` or `return None` in every branch of the function (including at the very bottom) which just seems annoying for an entry point function.